### DISCUSSION TITLE
Move table column scroll bar

### DISCUSSION THREAD

**Discussion by luigiPinnaAevo** [Budibase/budibase#9093] [Help]
Hello,

Is it possible to move the side of the table columns scroll bar from the bottom to the top of the table?

![table](https://user-images.githubusercontent.com/99037506/208409221-79b9c140-ba12-4027-92d7-5f54e57d9498.png)

Thanks :)

**Comment by melohagan** [comment]
Hey @luigiPinnaAevo 

I was able to hack this together, although I would question why you want to move the scrollbar to the top given it's just standard browser behaviour for tables.

Add an [Embed](https://docs.budibase.com/docs/custom-css#global-styling) component to your screen with the following content:
```html
<style>
.spectrum-Table {
  transform: rotateX(180deg) !important;
}
.spectrum-Table-cell {
  transform: rotateX(180deg) !important;
}
.spectrum-Table-headCell {
  transform: rotateX(180deg) !important;
}
</style>
```

**Result**
![Screenshot 2022-12-21 at 17 01 45](https://user-images.githubusercontent.com/101575380/208962317-94a9b341-c861-402b-a6d2-56489c15df6e.png)

In this case it's not ideal because the column headers are at the bottom of the table. 

Might be best going some sort of custom design with a Data Provider, Repeater and Containers if you really don't want the scrollbar to be along the bottom.

**Reply by luigiPinnaAevo** [reply]
Great, thank you for your support. 
Yes, I agree with you that it's the standard behaviour of browser tables. The client asked if it's possible to move the scrollbar to the top because sometimes the table is high and they have to use the browser scrollbar to see the bottom of the table. But when they move down they cannot see the table header.
